### ISSUE TITLE
Not enough image for UG 

### ISSUE THREAD

**Issue by tayruxin** [tayruxin/pe#18]
Currently, the UG only shows the message box. However for certain features it will be better to show the entire UI such as this feature. 

![image.png](https://raw.githubusercontent.com/tayruxin/pe/main/files/72d97f3b-50b7-41a7-ad94-8d214c080a81.png)

User may be unclear as to where the completed plan will be shown.

**Comment by nus-se-script** [comment]
[IMPORTANT!: Please do not edit or reply to this comment using the GitHub UI. You can respond to it using CATcher during the next phase of the PE]

# Team's Response
Thank you for your feedback!

We believe that the successful output message is sufficient in informing the user that the complete-plan command is successful.

Moreover, when a person creates a new Plan (which must be done before complete-plan), he would have seen the completion status under his Plan which will be intuitively updated when he executes a complete-plan command.

# Items for the Tester to Verify

## :question: Issue response

Team chose [`response.Rejected`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your reason]

<catcher-end-of-segment><hr>
